addons/zeus/functions/fnc_moduleSimulation.sqf

```diff
@@ -0,0 +1,28 @@
+/*
+ * Author: Fisher
+ * Toggle Simulation on object.
+ *
+ * Arguments:
+ * 0: The module logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+ 
+ #include "script_component.hpp"
+ 
+ params ["_logic"];
+ 
+ if !(local _logic) exitWith {};
+
+ private _object = attachedTo _logic;
+ if (isNull _object) then {
+    [LSTRING(NoObjectSelected)] call EFUNC(common,displayTextStructured);
+ } else {
+    private _simulationEnabled = simulationEnabled _object;
+    [[_object, (!_simulationEnabled)],"enableSimulationGlobal",false] call BIS_fnc_MP;
```

Comment:
that is not what @jonpas and I meant. Should probably be restored to what it was if the other comment regarding the modules locality is solved.